src/app/app.module.ts

```diff
@@ -34,6 +34,7 @@ import { LocalApiProvider } from '../providers/local-api/local-api.provider';
 import { SecureStorageProvider } from '../providers/secure-storage/secure-storage';
 import { StorageApiProvider } from '../providers/storage-api/storage-api.provider';
 import { WalletProvider } from '../providers/wallet/wallet.provider';
+import { SkyHeaderComponent } from '../components/sky-header/sky-header';
```

Comment:
Within our angular repo's we don't prefix components with Sky, just HeaderComponent will work